content/specific-skill-success-criteria/file-and-directory-naming/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+_db_id: 
+content_type: topic
+ready: true
+title: Naming modules and packages 
+---
+
+## Naming modules and packages 
+
+Files are refered to as modules and directories as folders or packages.
+
+Naming conventions are rules which enable the titling of electronic and physical folders, documents and records in a consistent and logical way.
+This ensures that the correct records can be located, identified and retrieved from a filing system in a timely fashion, and that they are stored 
+in an appropriate secure location. 
+
+## Benefits of naming conventions
+
+Naming records consistently, logically and in a predictable way will distinguish similar records from one another at a glance, and by doing so will 
+facilitate the storage and retrieval of data. Through consistency and the application of logical standards, we benefit from secure storage, and the 
+ability to locate and access information.
+
+Ideally,the best time to think about how to name and structure the documents and directories you create is at the start of a project. 
+
+## Note do all programmers
+
+New modules and packages (including third-party frameworks) should adhere to the recommended naming conventions, but if a library already exists that 
+has a different style, internal consistency should take precedence. 
+
+## How should I name my PYTHON modules and packages?
+
+Modules should have short, all-lowercase names and underscores can be used to improves readability. 
+
+Packages should also have short, all-lowercase names, although the use of underscores is discouraged.
+
+#### NO:
+
+    TASK/
+        1task.py
+
+    passwordChecker/
+        add-logger.py
+
+    Standard/
+        snake case.py
+        keepItShortAndmeaningfull.py
+
+#### YES:
+    task/
+        task_1.py
```

Comment:
Maybe add another task here, people might think that for example in the coding challenges they need to have each task in a separate folder (they already do this sometimes). Something like this clearly shows that we don't want each task in its own folder:
```
tasks/
  task_1.py
  task_2.py
```